modules/dictionary.lobster

```diff
@@ -69,5 +83,29 @@ class dictionary<K,V>:
                 c = c.next
         return keys, vals
 
+    def get_keys() -> [K]:
```

Comment:
If we're going to have these two methods, lets delete `get_all()`

Reply:
ok, will do